crates/notion-core/src/session.rs

```diff
@@ -182,22 +183,20 @@ impl Session {
         self.event_log.add_event_error(activity_kind, error)
     }
 
-    // send the events from this session to the monitor
-    pub fn send_events(&mut self) {
-        let command = self.events_command();
-        self.event_log.send_events(command)
-    }
-
-    // get the .notion.events_plugin string from package.json
-    pub fn events_command(&self) -> Option<String> {
-        self.project
-            .as_ref()
-            .and_then(|project| project.manifest().events_plugin.as_ref())
-            .map(|plugin| plugin.to_string())
-    }
-
     pub fn exit(mut self, code: i32) -> ! {
-        self.send_events();
+        match publish_plugin(&self.config) {
```

Comment:
I think it might be better to push this up to eagerly read the config file when creating the `Session`, since it's better to get an eager error about an invalid config file. But I'm filing that as a separate issue (#108).